src/servers/http_server.cc

```diff
@@ -35,10 +35,10 @@
 #include <algorithm>
 #include <list>
 #include <thread>
-#include "triton/common/constants.h"
-#include "triton/common/logging.h"
 #include "classification.h"
 #include "data_compressor.h"
+#include "triton/common/constants.h"
+#include "triton/common/logging.h"
```

Comment:
Looks like these are already included in the header "http_server.h"

Reply:
Good catch! Removed.